app/views/incomes/HistoricIncomeCalculationView.scala.html

```diff
@@ -15,123 +15,142 @@
+                        @{f"${historicIncomeCalculationViewModel.payments.last.taxAmountYearToDate}%,.2f"}
+                    </td>
+                    <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-bold">
+                        @{f"${historicIncomeCalculationViewModel.payments.last.nationalInsuranceAmountYearToDate}%,.2f"}
+                    </td>
+                </tr>
+                </tbody>
+            </table>
 
-                            </div>
-                        }
-                        case _ => {
-                            <div class="subsection subsection--wide">
-                                <p>@messages("tai.income.calculation.noRtiDataPreviousYear",Dates.formatDate(TaxYear(historicIncomeCalculationViewModel.taxYear.year).end))</p>
-                            </div>
+            <div id="eyu-messages" class="section flush--bottom soft--bottom">
+                <div class="eyu-multi-messages">
+                    @if(historicIncomeCalculationViewModel.endOfTaxYearUpdateMessages.size == 1) {
+                    <p class="govuk-body">
+                        @messages("tai.income.calculation.eyu.summary.single",
+                        historicIncomeCalculationViewModel.employerName.getOrElse("")) </p>
+                    <p class="govuk-body">@historicIncomeCalculationViewModel.endOfTaxYearUpdateMessages(0)</p>
+                    }
+                    @if(historicIncomeCalculationViewModel.endOfTaxYearUpdateMessages.size > 1) {
+                    <p class="govuk-body">@messages("tai.income.calculation.eyu.summary.multi",
+                        historicIncomeCalculationViewModel.employerName.getOrElse(""))</p>
+                    <ul id="eyu-multi-messages-bullets" class="govuk-list govuk-list--bullet" style="padding-left: 3%">
+                        @for(eyuMessage <- historicIncomeCalculationViewModel.endOfTaxYearUpdateMessages){
+                        <li>@eyuMessage</li>
                         }
+                    </ul>
                     }
-                    </div>
                 </div>
             </div>
+
         </div>
+        }
+        case _ => {
+            <div class="subsection subsection--wide">
+                <p class="govuk-body">
+                    @messages("tai.income.calculation.noRtiDataPreviousYear",Dates.formatDate(TaxYear(historicIncomeCalculationViewModel.taxYear.year).end))</p>
+            </div>
+        }
+        }
+        <div class="print-link-container hide-for-print">
+            <a id="print-link-btn" href="@routes.YourIncomeCalculationController.printYourIncomeCalculationHistoricYears(historicIncomeCalculationViewModel.taxYear,historicIncomeCalculationViewModel.employmentId).toString" class="govuk-link govuk-body-s gem-c-print-link-button" data-module="print-link">@Messages("tai.label.print")</a>
+        </div>
```

Comment:
has the print page been cleared with design?

Reply:
there's another ticket to implement the print so I did something in between: allow print using the previous controller while using the print button specified in the prototype